### ISSUE TITLE
Resolving of dependencies inside sub directories fails clientside

### ISSUE THREAD

**Issue by pma-pj** [altmp/altv-issues#1579]
### Description of the problem

[Since alt:V 10.0 you can also use resources in subdirectories.](https://docs.altv.mp/articles/configs/server.html)
In order to depend on resources placed inside sub directories, the sub-dir must be part of the resource identifier/name - **serverside**:
_some resource.cfg_
```
deps: [
    "subdir/resourceName"
]
```
Apparently the client cannot resolve those sub directories and fails to load the resource.

### Reproduction steps

Simply create a client resource that depends on a sub-dir resource:

_client-a/resource1/resource.cfg_
```
client-type: "js",
client-main: "empty.js",
deps: [
    "client-b/resource2"
]
```
_client-b/resource2/resource.cfg_
```
client-type: "js",
client-main: "empty.js"
```
_server.cfg_
```
resources: [
    "client-a/resource1",
    "client-b/resource2"
]
```
[Example config and resources](https://github.com/altmp/altv-issues/files/9219873/serverconfig.and.resources.zip), requires js-module.

### Expected behaviour

Resource loading shouldn't fail…

### Additional context

Output:
```
Loading resource resource1
Loading resource client-b/resource2
[Error] Failed to load resource File not found
[Error] Failed to load resource resource1 : Failed to start dependency
[Error] Failed to load resource resource1
Loading resource resource2
Loaded resource resource2
```

The sub-dir is only required if the resource hasn't already been loaded by the server.
If it has been loaded (correct order of resources in server.cfg) the server won't complain about the missing sub-dir and the client will load the resource successfully.

### Operating system

neon 20.04

### Version

release/12.7, dev/13.0-dev3

**Comment by deluvas1911** [comment]
confirmed

**Comment by C0kkie** [comment]
Will be fixed in release/12.9
